File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Info.java

package com.dinodevs.greatfitwatchface.theme.bin;

import java.util.HashMap;
import java.util.Map;

public class Info {

    private Integer deviceId;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Integer getDeviceId() {
        return deviceId;
    }

    public void setDeviceId(Integer deviceId) {
        this.deviceId = deviceId;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Info.kt
package com.dinodevs.greatfitwatchface.theme.bin

import java.util.*

class Info {
    var deviceId: Int? = null
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}